src/main/java/com/zendesk/maxwell/replication/BinlogConnectorReplicator.java

```diff
@@ -623,10 +650,22 @@ public RowMap getRow() throws Exception {
 		}
 	}
 
+	private String getXid(String sql) {
```

Comment:
`Xid` generally refers to something else in maxwell, maybe we call this `XAid` or something?

Reply:
em....As your wish!